spec/watcher_spec.rb

```diff
@@ -99,7 +99,7 @@
   end
 
   context "#listen_ignore_paths" do
-    let(:ignored) { subject.listen_ignore_paths(options) }
+    let(:ignored) { subject.send(:listen_ignore_paths, options) }
```

Comment:
@DirtyF Today I Learned™

Reply:
yeah, I just read about that too and about the whole debate on whether or not you should test private methods, so once again I leave it to the clean coders if this should be refactored or not.